### ISSUE TITLE
Blank selected column at the end of the sheet adds extra stage

### ISSUE THREAD

**Issue by theduncanclark** [kiln/template-horserace#82]
Not super urgent but I assume the intended behaviour is that we ignore trailing blank columns entirely. A Google News Lab user got confused when an empty column she didn't realise she had selected (I assume) added an extra stage causing the tie-break not to fire.

**Comment by daanlouter** [comment]
Though it might be nice to show roughly how much time is ahead? Eg. Premier League table with the remaining matches visible?

**Comment by theduncanclark** [comment]
True, but wouldn't that kind of thing usually have header rows so wouldn't be blank?

**Comment by daanlouter** [comment]
I see. Have tried to fix this, just now. But requires a quite a rewrite, so moving on for now.
